cmd/release-notes/README.md

```diff
@@ -46,6 +50,44 @@ You can also generate the raw notes data into JSON. You can then use a variety o
 ]
 ```
 
+if you would like to debug a run, use the `-debug` flag:
+
+```bash
+$ export GITHUB_TOKEN=a_github_api_token
+$ release-notes \
+  -start-sha 02dc3d713dd7f945a8b6f7ef3e008f3d29c2d549 \
+  -end-sha   23649560c060ad6cd82da8da42302f8f7e38cf1e \
+  -debug 
+
+level=debug timestamp=2019-07-30T04:02:43.8453116Z caller=notes.go:445 msg=################################################
+level=info timestamp=2019-07-30T04:02:43.8454168Z caller=notes.go:446 msg="[1/1679 - 0.06%]"
+level=debug timestamp=2019-07-30T04:02:43.8454701Z caller=notes.go:447 msg="Processing commit" func=ListCommitsWithNotes sha=23649560c060ad6cd82da8da42302f8f7e38cf1e
+level=debug timestamp=2019-07-30T04:02:44.3711956Z caller=notes.go:464 msg="Obtaining PR associated with commit sha '23649560c060ad6cd82da8da42302f8f7e38cf1e'." func=ListCommitsWithNotes prno=80301 prbody="**What type of PR is this?**\r\n> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:\r\n>\r\n> /kind api-change\r\n> /kind bug\r\n\r\n/kind cleanup\r\n\r\n> /kind design\r\n> /kind documentation\r\n> /kind failing-test\r\n> /kind feature\r\n> /kind flake\r\n\r\n**What this PR does / why we need it**:\r\n\r\nBased on the feedback from https://docs.google.com/document/d/1g5Aqa0BncQGRedSJH0TJQWq3mw3VxpJ_ufO1qokJ1LE we have decided to rename the `preferred` policy of the `TopologyManager` to `best-effort`.  The reasoning for this is outlined in the document.\r\n\r\nSince this change is coming before the `TopologyManager` was ever part of a release, this does not introduce a user-facing change.\r\n\r\n**Does this PR introduce a user-facing change?**:\r\n<!--\r\nIf no, just write \"NONE\" in the release-note block below.\r\nIf yes, a release note is required:\r\nEnter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string \"action required\".\r\n-->\r\n```release-note\r\nNONE\r\n```"
+level=debug timestamp=2019-07-30T04:02:44.3716249Z caller=notes.go:497 msg="Excluding notes for PR based on the exclusion filter." func=ListCommitsWithNotes filter="(?i)```(release-note\\s*)?('|\")?(none|n/a)?('|\")?\\s*```"
+...
+```
+
+## Options
+
+
+
+| Flag             | Env Variable    | Default Value | Required | Description                                             |
```

Comment:
Let's clean up the table formatting here (remove unnecessary space/extra hyphens) e.g.,:

| h1 | h2 | h3 |
| --- | --- | --- |
| thing1 | thing2 | thing3 |

See here: https://help.github.com/en/articles/organizing-information-with-tables#creating-a-table

Reply:
@justaugustus - all requested changes implemented. 

Line 32 in `cmd/release-notes/main.go` confused me - `go fmt` thought it was fine - turned out to be tabs vs spaces ... 😄